packages/tx-construction/src/tx-builder/OutputBuilder.ts

```diff
@@ -116,6 +118,12 @@ export class TxOutputBuilder implements OutputBuilder {
     return this;
   }
 
+  handle(output: Handle): OutputBuilder {
+    // if there is more than one handle, then maybe we should just ask for all of it
```

Comment:
See

Reply:
[f293dc8313](https://github.com/input-output-hk/cardano-js-sdk/pull/732/commits/f293dc8313a0c07bc570e18d025412315abc45ef) should solve this